zinc/src/sbt-test/source-dependencies/issue-810/test

```diff
@@ -0,0 +1,9 @@
+> compile
+
+> checkAPI core.A -513038468
```

Comment:
Unfortunately the hash is not stable between java8 and java 11 :(

Reply:
@eed3si9n any suggestion on how to write the assertion? other than this the PR is ready for review...

Reply:
If the values are not stable, that sounds like a bug we should address on its own.
But regardless, if it isn't then maybe it's not a good thing to test on. Can you say something like `A.b` exists?

Reply:
> If the values are not stable, that sounds like a bug we should address on its own.

Yeah I started to check why the hashes were not stable, and is due to some inherited methods from java.lang.Object having different annotations since java 9 (`@HotSpotIntrinsicCandidate` https://bugs.openjdk.java.net/browse/JDK-8076112). This can be seen by comparing the debug logs

```
java 8
[debug] [diff] ^inherited^ @scala.this#native( = ()) def hashCode(): scala.this#Int

java 11
[debug] [diff] ^inherited^ @scala.this#native( = ()) @jdk.internal.this#HotSpotIntrinsicCandidate( = ()) def hashCode(): scala.this#Int
```

Even if we ignored this annotation, the order of the methods is not exactly the same which would most likely also affect the hash...


> But regardless, if it isn't then maybe it's not a good thing to test on. Can you say something like `A.b` exists?

I am going to try that...